CONTRIBUTING.md

```diff
@@ -0,0 +1,130 @@
+# Contributing
+
+You are here to help on `bpmn-visualization-examples`? Awesome, feel welcome and read the following guidelines in order to know how to contribute, to ask questions and to make `bpmn-visualization` such a great tool.
+
+All members of our community are expected to follow our [Code of Conduct](https://github.com/process-analytics/.github/blob/main/CODE_OF_CONDUCT.md). Please make sure you are welcoming and friendly in all of our spaces.
+
+## Contributions 
+
+There are many ways to contribute:
+
+- update the [BPMN files](./bpmn-files) used for our test 
+- submitting bug reports and [new examples](./examples)in the [Github Issues](https://github.com/process-analytics/bpmn-visualization-examples/issues/new)
+- [improving](CONTRIBUTING.md#code-and-documentation-changes) the README
+
+## Code and documentation changes guidelines
+
+For all contributions, please respect the following guidelines:
+
+1. If you've noticed a bug or have a feature request, let us know in the [GitHub Issue tracker](https://github.com/process-analytics/bpmn-visualization-examples/issues/new )! So we can confirm the bug or approve your feature, and provide feedback, before starting to code :slightly_smiling_face:
+
+2. Do the changes in your own [fork](CONTRIBUTING.md#fork--create-a-branch) of the code
+
+3. Do not commit changes to files that are irrelevant to your feature or bugfix (eg: `.gitignore`).
+
+7. Open a [GitHub Pull Request](CONTRIBUTING.md#open-a-pull-request) with your patches. (**1** pull request = **1** feature or bug)
+   We will review your contribution and respond as quickly as possible. Keep in mind that this is an open source project, and it may take us some time to get back to you. Your patience is very much appreciated.
+
+8. If this is your 1st Pull Request, sign the [Contributor License Agreement](CONTRIBUTING.md#sign-the-contributor-license-agreement)
+
+9. Be willing to accept criticism and work on improving your code. 
+
+**Working on your first Pull Request?** You can learn how from this *free* series [How to Contribute to an Open Source Project on  GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github)    
+
+### Fork & create a branch
+
+[Fork bpmn-visualization-examples](https://help.github.com/articles/fork-a-repo) and create a branch with a descriptive name. 
+
+A good branch name would be (where issue #25 is the ticket you're working on): **25-annotations_to_tasks**
+
+```sh
+git checkout -b 25-annotations_to_tasks
+```
+
+### IDE configuration
+To know how to configure your development environment, see [IDE configuration](https://github.com/process-analytics/bpmn-visualization-js/blob/master/docs/development/ide-configuration.md)
+
+
+### Adding new example
+Make the examples use the latest version of the lib.
+Do not forget to update file /demo/index.html with the Latest Release information.
+Tests that the examples work with the new lib version.
```

Comment:
:warning: wrong tests, usage of the latest lib version is part of the lib release
If we want to add this paragraph, explain why and how add new examples

Reply:
Bad copy/paste :(

Reply:
Fixed

Reply:
thx